p/param/build_info.json

```diff
@@ -0,0 +1,16 @@
+{
+  "maintainer": "mohitp333",
+  "package_name": "param",
+  "github_url": "https://github.com/holoviz/param.git",
+  "version": "v2.1.0",
+  "default_branch": "main",
+  "package_dir": "p/param",
+  "build_script": "param_ubi_9.3.sh",
+  "validate_build_script": "true",
+  "use_non_root_user": "false",
+  "docker_build": "true",
+  "v2.*.*": {
+    "dir": "v2.1.0_ubi_9",
+    "build_script": "param_ubi_9.3.sh"
+  }
```

Comment:
Change this to v*.*.* for now so any version build will get dir value

Reply:
Required changes are done.